chatcommands.lua

```diff
@@ -286,6 +286,30 @@ minetest.register_chatcommand("area_open", {
 })
 
 
+if areas.factions_available then
+	minetest.register_chatcommand("area_faction_open", {
+		params = "<ID>",
+		description = "Toggle an area open (anyone can interact) or closed to your faction",
```

Comment:
As a player: what does this mean?
Something like this might be more helpful: (dunno)
`Toggle an area open/closed for members in your faction.`

Reply:
Description has been updated, thanks